cmd/s3.go

```diff
@@ -26,24 +26,29 @@ var s3Cmd = &cobra.Command{
 }
 
 func getS3(cmd *cobra.Command, args []string) error {
-	ctx := context.TODO()
-	cfg, err := config.LoadDefaultConfig(ctx)
-	if err != nil {
-		log.Fatal(err)
-	}
 
-	// Create an Amazon S3 service client
-	client := s3.NewFromConfig(cfg)
+	ctx := context.TODO()
+	client := pkg.S3Client(ctx)
 
 	out, err := client.ListBuckets(ctx, &s3.ListBucketsInput{})
```

Comment:
@riteshsonawane1372 also get this function in pkg as you will add struct for s3.

it will look like,

```
s3List,err := pkg.ListBuckets(....) 
```

Reply:
should I do it `` S3.ListBuckets`` ?

Reply:
yeah that works too. with this we will be keeping all the AWS API calls under pkg directory and cmd only for commands and printing stuff